frontend/src/components/ForgotPasswordPage/ForgotPasswordPage.css

```diff
@@ -0,0 +1,81 @@
+@import "../../config/colors.css";
+@import url('https://fonts.googleapis.com/css?family=Rubik&display=swap');
+
+#forgotPasswordBox {
+    width: 610px;
+    height: 420px;
+    margin: auto;
+    left: 515px;
+    top: 407px;
+
+    box-sizing: border-box;
+
+    background: var(--white-gray);
+    margin: auto;
+    margin-top: 50vh;
+    transform: translateY(-50%);
+    box-shadow: 0px 8px 21px var(--box-shadow);
+    padding: 55px 85px 68px 85px;
+}
+
+#forgotPasswordText {
+    font-family: Rubik;
+    font-weight: 600;
+    font-size: 36px;
+    line-height: 40px;
+    margin: 0px 0px 34px 0px;
+    text-align: left;
+
+    color: var(--black);
+}
+
+.forgotPasswordMessage {
+    font-weight: 1000;
+    font-size: 17px;
+    line-height: 25px;
+    margin: 0px 0px 34px 0px;
+    text-align: left;
+
+    color: var(--black);
+}
+
+.emailInput {
+    font-family: Rubik;
+    font-weight: bold;
+    font-size: 16px;
+    line-height: 25px;
+    margin: 0px 0px 5px 0px;
+    text-align: left;
+
+    color: var(--black);
+}
+
+#sendButton {
+    background: var(--primary);
+    border: none;
+    font-family: Rubik;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 21px;
+    margin: 26px 0px 12px 0px;
+    height: 51px;
+    width: 100%;
+    color: var(--white);
+}
+
+#sendButton:hover{
+    cursor :pointer;
+    border: 3px solid rgba(2, 5, 12, 0.15)
+}
+
+.inputBox {
+    font-family: Rubik;
+    box-sizing: border-box;
+    height: 43px;
+    width: 100%;
+    border: 1px solid var(--input-box);
+    padding: 0px;
```

Comment:
```suggestion
    padding: 9px;
```
Just adding some padding:
<img width="554" alt="Screen Shot 2022-01-17 at 7 27 57 PM" src="https://user-images.githubusercontent.com/42549561/149865885-e4c7d631-350f-445c-b9ee-69a0c9898cd1.png">